src/components/ClaimSudt/index.tsx

```diff
@@ -1,60 +1,15 @@
-import { CkitProvider, MintOptions, MintSudtBuilder, predefined } from "@ckitjs/ckit";
-import { Secp256k1Signer } from "@ckitjs/ckit/dist/wallets/Secp256k1Wallet";
 import { notification } from "antd";
 import { useLightGodwoken } from "../../hooks/useLightGodwoken";
-import { BI } from "@ckb-lumos/lumos";
-import { CKB_USDC_ISSUER_PRIVATE_KEY } from "../../config";
-
-const claim = async ({
-  recipientAddr,
-  CKBIndexerUrl,
-  CKBRpcUrl,
-}: {
-  recipientAddr: string;
-  CKBIndexerUrl: string;
-  CKBRpcUrl: string;
-}) => {
-  const provider = new CkitProvider(CKBIndexerUrl, CKBRpcUrl);
-  await provider.init(predefined.Aggron);
-
-  const { SECP256K1_BLAKE160 } = provider.config.SCRIPTS;
-  const issuerSigner = new Secp256k1Signer(CKB_USDC_ISSUER_PRIVATE_KEY, provider, {
-    code_hash: SECP256K1_BLAKE160.CODE_HASH,
-    hash_type: SECP256K1_BLAKE160.HASH_TYPE,
-  });
-
-  const recipients: MintOptions["recipients"] = [
-    {
-      recipient: recipientAddr,
-      additionalCapacity: "0",
-      amount: BI.from(1000).mul(BI.from(10).pow(18)).toString(),
-      capacityPolicy: "createCell",
-    },
-  ];
-
-  const unsigned = await new MintSudtBuilder({ recipients }, provider, await issuerSigner.getAddress()).build();
-  const mintTxHash = await provider.sendTransaction(await issuerSigner.seal(unsigned));
-  return mintTxHash;
-};
 
 export const ClaimSudt: React.FC = () => {
   const lightGodwoken = useLightGodwoken();
-  const config = lightGodwoken?.provider.getLightGodwokenConfig();
 
   const claimSudt = async () => {
     if (!lightGodwoken) {
-      notification.error({ message: "Please connect wallet first" });
-      return;
-    }
-    if (!config) {
-      throw new Error("can not find light godwoken config");
+      throw new Error("LightGodwoken Not Found!");
```

Comment:
```suggestion
      notification.error({ message: "Please connect wallet first" });
      return
```
Interaction change in New UI, we need show error message to user